dist_tables/ddl.rst

```diff
@@ -0,0 +1,73 @@
+.. _ddl:
+
+Creating Distributed Tables (DDL)
+#################################
+
+.. _dist_tables_dataset:
+
+Example Dataset
```

Comment:
Do we need this as a top-level section? https://citus-doc-preview-pr-157.herokuapp.com/dist_tables/ddl.html#example-dataset. It feels a bit odd that the first subsection in Creating Distributed Tables is 'Example Dataset'.